lib/actions/user.js

```diff
@@ -293,3 +296,26 @@ export function verifyPhoneNumber (code) {
     }
   }
 }
+
+/**
+ * Check itinerary existence for the given monitored trip.
+ */
+export function checkItineraryExistence (trip) {
+  return async function (dispatch, getState) {
+    const { accessToken, apiBaseUrl, apiKey } = getMiddlewareVariables(getState())
+    const requestUrl = `${apiBaseUrl}${API_MONITORED_TRIP_PATH}${API_MONITORED_TRIP_CHECK_SUBPATH}`
```

Comment:
I don't think `API_MONITORED_TRIP_CHECK_SUBPATH` needs to be in a constant if it's only used one time. Having constants far away from the place they're used can make things hard to read.

Reply:
Sure. Updated in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/846892ab816dc32ccd6688b06042f030225c453a